### ISSUE TITLE
Pre-work Checklist: Developer: Travis Bennett

### ISSUE THREAD

**Issue by one2code** [hackforla/website#4335]
### Prerequisite
We are looking forward to having you on our team. Please make sure to attend the general Hack for LA onboarding to get the process started https://meetup.com/hackforla/events.

### Overview
As a new developer on the HfLA website team, fill in the following fields as you complete each onboarding item.

### Special Notes
1. It may take you a few weeks to finish this issue, because part of it is learning how to provide updates on issues that take more than a week to complete.    Please keep this issue open until you have been able to do all the steps.  
2. Usually we don't want to you have more than one issue assigned to you at a time, this issue is the exception, because it is instructions on how to work on other issues.
3. The action items listed below should mostly be worked on in a sequential order. However, you don't have to wait on one if you can proceed with the others, For instance, you don't have to wait for attending a weekly meeting before setting up your dev environment.  

### Action Items

- [x] Add yourself to the #hfla-site and #hfla-site-pr Slack channels
- [x] Share your GitHub handle and Gmail address in the hfla-site-onboarding slack channel so you can be added to the Google Drive (alternative to go to meeting and direct message a merge team member at the [meeting](https://github.com/hackforla/website/wiki/Meetings-and-Agendas
))
  - [x] To find contact information for the merge team members and technical leads, please take a look at our [Meet the Team wiki page](https://github.com/hackforla/website/wiki/Meet-the-Team)
- [x] Also, confirm with a merge team member or a technical lead that they have added you to the Google Calendar invites for our Zoom meetings
- [x] (Once added to the Drive) Add yourself to the [team roster](https://docs.google.com/spreadsheets/d/11u71eT-rZTKvVP8Yj_1rKxf2V45GCaFz4AXA7tS_asM/edit#gid=0)
- [x] After you have finished adding yourself to the roster, let a merge team member or a technical lead know you have added yourself to the roster and would like to be added to the `website-write` and `website` teams on GitHub
- [x] Once added to the website-write team:
  - [x] Self Assign this issue (gear in right side panel)
  - [x] Add the "role: front end" or "role: back end" or both label(s) to this issue and remove the "role missing" label (gear in right side panel)
  - [x] Add this issue to the Project Board under the Projects section (gear in right side panel)
- [x] Signup for a [Figma](https://Figma.com) account
- [x] Attend weekly team meetings:
  - [x] Developer (front-end/back-end) weekly team meeting, Tuesdays 7-8pm PST
  - [x] (Optional) Office Hours, Thursdays 7-8pm PST
  - [x] All team meeting (UX, Development, Product), Sunday 10am-12pm PST
- [x] Note: There are no meetings on the 1st-7th of every month.
- [x] Note regarding weekly team meeting requirements: All website team members are required to attend at least 1 team meeting in a week (held on Tuesdays, Thursdays and Sundays). In case, you are unable in any given week, you should reach out to the tech leadership team. Exceptions to this requirement may be provided on a case-by-case basis. Also, please let the tech leadership team know (through a slack message in the hfla-site channel as well as an @ mention in a comment of the issue that you would be working on) if you are planning to take a week off or a longer vacation.
- [x] Complete steps 1.1 - 1.6 in [Part 1: Setting up the development environment within Contributing.md](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#part-1-setting-up-the-development-environment)
  - [x] OPTIONAL: If you run into any issues, use [4.1 How do I ask for help within Contributing.md](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#41-what-do-i-do-if-i-need-help) as a visual example to get a hold of our [HfLA website leadership team](https://github.com/hackforla/website/projects/7#card-69730135). That way we can help you resolve any set up issues immediately.
- [x] Read section 2.1 - 2.4 in [Part 2: How the Website team works with GitHub issues within Contributing.md](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#part-2-how-the-website-team-works-with-github-issues)
  - [x] Follow the steps in section [2.3 Where can I find GitHub issues to work on?](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#23-where-can-i-find-github-issues-to-work-on) to assign yourself your first issue under the Project Board's Prioritized Backlog column and use [2.7 Working on a Issue within Contributing.md](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#27-working-on-an-issue) to start working on your issue
  - [x] Read [2.6 What to do when you need to stop mid issue](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#26-what-to-do-when-you-need-to-stop-mid-issue)
- [x] Once you take a good first issue, you will provide estimates and progress reports. Also, once you finish providing your "Estimate" action item below, please answer the question in the "Time spent so far" action item (also below).
   - [x] Estimate: Copy the below and put it in the "good first issue" that you picked.
 Check this box when you have completed this task so that we can identify if you understood the instructions and know what to do on all subsequent issues upon assignment. (Note: Please provide estimates on all the issues that you pick up, going forward.)
    ```
   Availability for this week:
   My estimated ETA for completing this issue:
    ```
   - [x] Time spent so far: Copy the question below into a new comment below (in this pre-work issue only) and answer it. This is just to get feedback on how long it took you to get to this point on this pre-work. There is no right or wrong answers. There is no judgement. It is ok if you take a long time or if you do it really fast or at any pace.
    ```
    How many hours did it take you to finish the pre-work up to and including adding your initial ETA and availability for your good first issue, including attending your first meetings?
    ```
   - [x] Progress Reports: Copy the below and put it in the issue once you have been assigned to the issue at least 5 days (we check weekly on Fridays), or sooner if you have something to report.  If you finish this issue before 5 days are reached, Yeah!!, do it on your next issue. **This update should be done every week for every issue that you are assigned to**. The checkbox here is meant for us to see if you understood the instructions when you end up doing your first weekly progress update.
    ```
    Provide Update
    1. Progress
    2. Blockers
    3. Availability
    4. ETA
    ```
- [x] Read sections 3.1.a - 3.1.c in [3.1 How to make a pull request](https://github.com/hackforla/website/blob/2e3c45fede99e61f479548ccca3f0dda83e351db/CONTRIBUTING.md#31-how-to-make-a-pull-request) to learn more about how to make a pull request (PR) for the issue that you are working on and how to make changes to your PR if changes are requested by the reviewer
- [x] Additional notes:
   - [x] Please don't forget to review pull requests whenever you are able to.  The [How to review Pull Requests](https://github.com/hackforla/website/wiki/How-to-review-pull-requests) guide will teach you how to review pull requests.
   - [x] Please work on only one issue at a time and wait until your pull request is merged before picking up another issue.
- [x] Read and understand how we progress through issues. Then, you can check this off.
Progress through issues in the [prioritized backlog](https://github.com/hackforla/website/projects/7#column-7198257) only with increasing complexity in the following order:
  - Good first issue (one per person)
  - Good second issue (one per person)
  - Small (one per person, with some exceptions, see below)
  - Medium (you can work on more than one medium issue, but only one at a time)
  - Large (you can work on more than one large issue, but only one at a time)
    - The reasons for this progression are:
      - The issues start out as being prescriptive and become less so as you gain more experience by working through increasingly complex issues.
      - We are trying to teach you the team methodology through the issues themselves.
      - It ensures you understand what we expect and the quality of contributions.
    - You can work on back-to-back small issues if it meets the following criteria:
      - You are learning something new and need to work on an issue of a lesser complexity
      - Special request by a lead or pm
- [x] Read the [Start Here - Developers](https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website?node-id=8583%3A0) in Figma
- [x] Go familiarize yourself with the [Hack for LA Design System page in Figma](https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website?node-id=3464%3A3)  (where you can see components and their SCSS classes)
- [x] Once all tasks are completed, close this issue.  This issue will remain open until you get through your first progress report.  Once you have done that, we are confident you know how to keep the momentum going on your issue and keep your team informed.

### What should I do if I have a question about an issue I'm working on, and I haven't gotten a response yet?
- First, you should post the question or blocker as a comment on your assigned issue, so it can be easily referred to in the next bullet points.
- Then, add the label "Status: Help Wanted" so other developers can see it and potentially help answer your question. In addition, you will still need to post a Slack message or bring it up in meeting so we know you need help; see below for how to do that.
- Also, you can post your question on the hfla-site slack channel and link the issue you're working on, so other developers can see and respond.
- Lastly, you can add the issue to the "Development team meeting discussion items" column of the Project Board so that it can be addressed in the next development meeting. Please bring it during the meeting that you need help.

### Resources/Instructions
- [Contributing.md - Hack for LA](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md)
- [GitHub Project Board - Hack for LA](https://github.com/hackforla/website/projects/7)
- [GitHub Project Board Prioritized Backlog highlighted and sorted by Front End](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22role%3A+front+end%22)
- [GitHub Project Board Prioritized Backlog highlighted and sorted by Back End/DevOps](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22role%3A+back+end%2Fdevops%22)
- [Figma - Hack for LA](https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website)
- [Google Drive - Hack for LA website team](https://drive.google.com/drive/folders/1p76K0FgfiAWeIIEyoyJ_Iik8FVj8cBjT?usp=sharing)
- [Agenda / Notes - Dev Team Tuesday meeting](https://github.com/hackforla/website/issues/2010)
- [Agenda / Notes - All Team meeting](https://github.com/hackforla/website/issues/2027)
- [How to review Pull Requests](https://github.com/hackforla/website/wiki/How-to-review-pull-requests)

**Comment by ExperimentsInHonesty** [comment]
@one2code The next team meeting is this Sunday at 10am pst.  Or the one after that is the Tuesday meeting at 7pm pst.  Reach out in the #hfla-slack channel if you are having trouble setting up your dev environment.

**Comment by one2code** [comment]
**_How many hours did it take you to finish the pre-work up to and including adding your initial ETA and availability for your good first issue, including attending your first meetings?_**

Around 6-8 hours selecting a good first issue and posting my ETA. Blockers encountered primarily revolved around fully setting up Docker. I had to clear an additional 20GB of hard drive space to setup the GUI Docker application.  Overall timeframe to attend my first meeting? 3 weeks. I happened to have onboarded at the end of March, so the time period to complete up to my first meeting was more related to timing of coming onboard.

**Comment by HackforLABot** [comment]
## Activity Log: one2code
### Repo: https://github.com/hackforla/website

#####  ⚠ Important note: The bot updates this comment automatically - do not edit

- one2code opened: https://github.com/hackforla/website/issues/4335 at 03/28/2023, 09:19 PM PDT
- one2code assigned: https://github.com/hackforla/website/issues/4335 at 04/06/2023, 01:57 PM PDT
- one2code assigned: https://github.com/hackforla/website/issues/4354 at 04/13/2023, 09:08 PM PDT
- one2code unassigned: https://github.com/hackforla/website/issues/4354 at 04/13/2023, 11:59 PM PDT
- one2code assigned: https://github.com/hackforla/website/issues/4380 at 04/14/2023, 12:08 AM PDT
- one2code assigned: https://github.com/hackforla/website/issues/3524#issuecomment-1240946700 at 04/14/2023, 11:57 AM PDT
- one2code unassigned: https://github.com/hackforla/website/issues/3524#issuecomment-1240946700 at 04/14/2023, 11:57 AM PDT
- one2code assigned: https://github.com/hackforla/website/issues/4383 at 04/14/2023, 11:58 AM PDT
- one2code commented on: https://github.com/hackforla/website/issues/4380#issuecomment-1509915110 at 04/15/2023, 10:56 AM PDT
- one2code opened: https://github.com/hackforla/website/pull/4520 at 04/15/2023, 12:51 PM PDT
- one2code unassigned: https://github.com/hackforla/website/issues/4383 at 04/15/2023, 12:59 PM PDT
- one2code commented on: https://github.com/hackforla/website/pull/4520#issuecomment-1510445913 at 04/16/2023, 11:03 AM PDT
- one2code PR was merged: https://github.com/hackforla/website/pull/4520#event-9018434056 at 04/16/2023, 02:34 PM PDT
- one2code issue closed by PR 4520: https://github.com/hackforla/website/issues/4380#event-9018434749 at 04/16/2023, 02:34 PM PDT
- one2code assigned: https://github.com/hackforla/website/issues/4400 at 04/16/2023, 07:05 PM PDT
- one2code commented on: https://github.com/hackforla/website/issues/4400#issuecomment-1510589969 at 04/16/2023, 07:07 PM PDT
- one2code opened: https://github.com/hackforla/website/pull/4528 at 04/16/2023, 09:30 PM PDT
- one2code PR was merged: https://github.com/hackforla/website/pull/4528#event-9031100177 at 04/17/2023, 09:08 PM PDT
- one2code issue closed by PR 4528: https://github.com/hackforla/website/issues/4400#event-9031100259 at 04/17/2023, 09:08 PM PDT
- one2code commented on: https://github.com/hackforla/website/issues/4335#issuecomment-1515612317 at 04/19/2023, 07:09 PM PDT
- one2code assigned: https://github.com/hackforla/website/issues/3843#issuecomment-1499866166 at 04/20/2023, 06:25 AM PDT
- one2code commented on: https://github.com/hackforla/website/issues/3843#issuecomment-1516337227 at 04/20/2023, 06:31 AM PDT
- one2code commented on: https://github.com/hackforla/website/issues/3843#issuecomment-1526904211 at 04/27/2023, 07:47 PM PDT
- one2code closed as completed: https://github.com/hackforla/website/issues/4335#event-9120475701 at 04/27/2023, 07:49 PM PDT
- one2code commented on: https://github.com/hackforla/website/issues/3843#issuecomment-1536768291 at 05/05/2023, 01:52 PM PDT
- one2code commented on: https://github.com/hackforla/website/issues/3843#issuecomment-1542414695 at 05/10/2023, 08:32 AM PDT
- one2code opened: https://github.com/hackforla/website/pull/4661 at 05/12/2023, 06:55 PM PDT
- one2code commented on: https://github.com/hackforla/website/pull/4661#issuecomment-1550598025 at 05/16/2023, 07:30 PM PDT
- one2code opened: https://github.com/hackforla/website/pull/4688 at 05/17/2023, 05:57 PM PDT
- one2code closed: https://github.com/hackforla/website/pull/4661#event-9274744149 at 05/17/2023, 05:58 PM PDT
- one2code reopened: https://github.com/hackforla/website/pull/4661#event-9274744149 at 05/18/2023, 11:58 AM PDT
- one2code PR was closed: https://github.com/hackforla/website/pull/4661#event-9282894173 at 05/18/2023, 11:59 AM PDT
- one2code submitted review: https://github.com/hackforla/website/pull/4683#pullrequestreview-1433774626 at 05/18/2023, 09:56 PM PDT
- one2code commented on: https://github.com/hackforla/website/pull/4698#issuecomment-1555961148 at 05/20/2023, 10:51 AM PDT
- one2code submitted review: https://github.com/hackforla/website/pull/4698#pullrequestreview-1435482769 at 05/21/2023, 12:02 AM PDT
- one2code submitted review: https://github.com/hackforla/website/pull/4683#pullrequestreview-1438593723 at 05/22/2023, 10:49 PM PDT
- one2code commented on: https://github.com/hackforla/website/pull/4688#issuecomment-1559420378 at 05/23/2023, 06:51 AM PDT
- one2code commented on: https://github.com/hackforla/website/pull/4708#issuecomment-1560059549 at 05/23/2023, 01:10 PM PDT
- one2code submitted review: https://github.com/hackforla/website/pull/4708#pullrequestreview-1440884350 at 05/23/2023, 08:36 PM PDT
- one2code submitted review: https://github.com/hackforla/website/pull/4708#pullrequestreview-1442719782 at 05/24/2023, 01:42 PM PDT
- one2code submitted review: https://github.com/hackforla/website/pull/4725#pullrequestreview-1442970195 at 05/24/2023, 07:02 PM PDT
- one2code commented on: https://github.com/hackforla/website/pull/4726#issuecomment-1562307936 at 05/24/2023, 10:41 PM PDT
- one2code PR was merged: https://github.com/hackforla/website/pull/4688#event-9343513105 at 05/25/2023, 11:57 AM PDT
- one2code opened: https://github.com/hackforla/website/pull/4733 at 05/25/2023, 07:46 PM PDT
- one2code PR was merged: https://github.com/hackforla/website/pull/4733#event-9346034481 at 05/25/2023, 07:57 PM PDT
- one2code commented on: https://github.com/hackforla/website/pull/4747#issuecomment-1565176270 at 05/26/2023, 07:57 PM PDT
- one2code submitted review: https://github.com/hackforla/website/pull/4747#pullrequestreview-1447134137 at 05/26/2023, 08:27 PM PDT
- one2code submitted review: https://github.com/hackforla/website/pull/4747#pullrequestreview-1447728590 at 05/27/2023, 06:03 PM PDT
- one2code commented on: https://github.com/hackforla/website/pull/4726#issuecomment-1565804611 at 05/27/2023, 07:16 PM PDT
- one2code submitted review: https://github.com/hackforla/website/pull/4726#pullrequestreview-1447805971 at 05/27/2023, 07:46 PM PDT
- one2code assigned: https://github.com/hackforla/website/issues/3975#issuecomment-1565195706 at 05/27/2023, 08:26 PM PDT
- one2code commented on: https://github.com/hackforla/website/issues/3975#issuecomment-1565841497 at 05/27/2023, 08:33 PM PDT
- one2code submitted review: https://github.com/hackforla/website/pull/4726#pullrequestreview-1447849700 at 05/27/2023, 08:52 PM PDT
- one2code issue closed as completed: https://github.com/hackforla/website/issues/3843#event-9362451206 at 05/28/2023, 08:31 AM PDT
- one2code commented on: https://github.com/hackforla/website/pull/4754#issuecomment-1567910077 at 05/30/2023, 12:29 AM PDT
- one2code submitted review: https://github.com/hackforla/website/pull/4754#pullrequestreview-1450553645 at 05/30/2023, 02:50 AM PDT
- one2code commented on: https://github.com/hackforla/website/pull/4767#issuecomment-1573037525 at 06/01/2023, 07:12 PM PDT
- one2code commented on: https://github.com/hackforla/website/issues/3975#issuecomment-1574323760 at 06/02/2023, 02:15 PM PDT
- one2code submitted review: https://github.com/hackforla/website/pull/4767#pullrequestreview-1464139771 at 06/05/2023, 07:50 PM PDT
- one2code opened: https://github.com/hackforla/website/pull/4796 at 06/10/2023, 01:53 PM PDT
- one2code commented on: https://github.com/hackforla/website/pull/4817#issuecomment-1588116510 at 06/12/2023, 02:16 PM PDT
- one2code submitted review: https://github.com/hackforla/website/pull/4817#pullrequestreview-1475931981 at 06/12/2023, 03:52 PM PDT
- one2code submitted review: https://github.com/hackforla/website/pull/4817#pullrequestreview-1476202063 at 06/12/2023, 09:03 PM PDT
- one2code commented on: https://github.com/hackforla/website/pull/4833#issuecomment-1590356151 at 06/13/2023, 07:38 PM PDT
- one2code commented on: https://github.com/hackforla/website/pull/4832#issuecomment-1590356456 at 06/13/2023, 07:38 PM PDT
- one2code PR was merged: https://github.com/hackforla/website/pull/4796#event-9560042415 at 06/17/2023, 08:18 PM PDT
- one2code issue closed by PR 4796: https://github.com/hackforla/website/issues/3975#event-9560042438 at 06/17/2023, 08:18 PM PDT
- one2code submitted review: https://github.com/hackforla/website/pull/4833#pullrequestreview-1484964312 at 06/17/2023, 09:36 PM PDT
- one2code commented on: https://github.com/hackforla/website/pull/4832#issuecomment-1596525663 at 06/18/2023, 10:23 PM PDT
- one2code commented on: https://github.com/hackforla/website/pull/4832#issuecomment-1598052022 at 06/19/2023, 08:18 PM PDT
- one2code submitted review: https://github.com/hackforla/website/pull/4832#pullrequestreview-1489332130 at 06/20/2023, 05:49 PM PDT
- one2code commented on: https://github.com/hackforla/website/pull/4875#issuecomment-1605670075 at 06/24/2023, 10:57 AM PDT
- one2code submitted review: https://github.com/hackforla/website/pull/4875#pullrequestreview-1496772952 at 06/24/2023, 11:37 AM PDT
- one2code commented on: https://github.com/hackforla/website/pull/4878#issuecomment-1605768079 at 06/24/2023, 04:06 PM PDT
- one2code commented on: https://github.com/hackforla/website/pull/4879#issuecomment-1605839876 at 06/24/2023, 08:12 PM PDT
- one2code submitted review: https://github.com/hackforla/website/pull/4879#pullrequestreview-1497088682 at 06/25/2023, 07:02 AM PDT
- one2code submitted review: https://github.com/hackforla/website/pull/4878#pullrequestreview-1497098445 at 06/25/2023, 07:28 AM PDT
- one2code commented on: https://github.com/hackforla/website/pull/4882#issuecomment-1607657503 at 06/26/2023, 07:54 AM PDT
- one2code commented on: https://github.com/hackforla/website/pull/4879#issuecomment-1608209327 at 06/26/2023, 01:38 PM PDT
- one2code submitted review: https://github.com/hackforla/website/pull/4882#pullrequestreview-1499914638 at 06/26/2023, 09:22 PM PDT
- one2code submitted review: https://github.com/hackforla/website/pull/4879#pullrequestreview-1499921835 at 06/26/2023, 09:29 PM PDT
- one2code assigned: https://github.com/hackforla/website/issues/4475 at 06/27/2023, 02:14 AM PDT
- one2code commented on: https://github.com/hackforla/website/issues/4475#issuecomment-1609113855 at 06/27/2023, 02:18 AM PDT
- one2code commented on: https://github.com/hackforla/website/issues/4475#issuecomment-1610674971 at 06/27/2023, 09:17 PM PDT
- one2code opened: https://github.com/hackforla/website/pull/4896 at 06/28/2023, 12:21 AM PDT
- one2code submitted review: https://github.com/hackforla/website/pull/4832#pullrequestreview-1505380205 at 06/29/2023, 07:07 AM PDT
- one2code commented on: https://github.com/hackforla/website/pull/4899#issuecomment-1613266890 at 06/29/2023, 07:19 AM PDT
- one2code commented on: https://github.com/hackforla/website/pull/4896#issuecomment-1613286514 at 06/29/2023, 07:30 AM PDT
- one2code commented on: https://github.com/hackforla/website/pull/4902#issuecomment-1615352610 at 06/30/2023, 06:59 PM PDT
- one2code submitted review: https://github.com/hackforla/website/pull/4879#pullrequestreview-1509648034 at 07/02/2023, 10:50 AM PDT
- one2code commented on: https://github.com/hackforla/website/pull/4905#issuecomment-1620784724 at 07/04/2023, 02:59 PM PDT
- one2code commented on: https://github.com/hackforla/website/pull/4896#issuecomment-1620955932 at 07/04/2023, 08:16 PM PDT
- one2code commented on: https://github.com/hackforla/website/pull/4896#issuecomment-1622949664 at 07/05/2023, 09:17 PM PDT
- one2code commented on: https://github.com/hackforla/website/pull/4896#issuecomment-1626355160 at 07/07/2023, 04:22 PM PDT
- one2code commented on: https://github.com/hackforla/website/pull/4896#issuecomment-1627576891 at 07/08/2023, 07:22 PM PDT
- one2code submitted review: https://github.com/hackforla/website/pull/4905#pullrequestreview-1520946179 at 07/09/2023, 12:00 PM PDT
- one2code commented on: https://github.com/hackforla/website/pull/4896#issuecomment-1629894651 at 07/10/2023, 05:00 PM PDT
- one2code commented on: https://github.com/hackforla/website/pull/4936#issuecomment-1631745229 at 07/11/2023, 07:12 PM PDT
- one2code PR was merged: https://github.com/hackforla/website/pull/4896#event-9795862162 at 07/12/2023, 01:03 AM PDT
- one2code issue closed by PR 4896: https://github.com/hackforla/website/issues/4475#event-9795862474 at 07/12/2023, 01:03 AM PDT
- one2code commented on: https://github.com/hackforla/website/pull/4927#issuecomment-1635212536 at 07/13/2023, 08:24 PM PDT
- one2code submitted review: https://github.com/hackforla/website/pull/4927#pullrequestreview-1529557337 at 07/13/2023, 08:59 PM PDT
- one2code submitted review: https://github.com/hackforla/website/pull/4936#pullrequestreview-1531302726 at 07/14/2023, 11:39 PM PDT
- one2code commented on: https://github.com/hackforla/website/pull/4936#issuecomment-1637346041 at 07/16/2023, 09:05 PM PDT
- one2code submitted review: https://github.com/hackforla/website/pull/4936#pullrequestreview-1531905918 at 07/16/2023, 09:08 PM PDT
- one2code commented on: https://github.com/hackforla/website/pull/4953#issuecomment-1640893749 at 07/18/2023, 12:38 PM PDT
- one2code submitted review: https://github.com/hackforla/website/pull/4953#pullrequestreview-1536212764 at 07/18/2023, 08:55 PM PDT
- one2code commented on: https://github.com/hackforla/website/pull/5006#issuecomment-1643056606 at 07/19/2023, 08:41 PM PDT
- one2code submitted review: https://github.com/hackforla/website/pull/5006#pullrequestreview-1538379206 at 07/19/2023, 08:50 PM PDT
- one2code submitted review: https://github.com/hackforla/website/pull/5006#pullrequestreview-1540125617 at 07/20/2023, 02:35 PM PDT
- one2code submitted review: https://github.com/hackforla/website/pull/5105#pullrequestreview-1551301245 at 07/27/2023, 08:52 PM PDT
- one2code submitted review: https://github.com/hackforla/website/pull/4953#pullrequestreview-1557024399 at 08/01/2023, 06:35 AM PDT
- one2code submitted review: https://github.com/hackforla/website/pull/5144#pullrequestreview-1564070057 at 08/05/2023, 10:42 PM PDT
- one2code submitted review: https://github.com/hackforla/website/pull/5170#pullrequestreview-1570962630 at 08/09/2023, 08:42 PM PDT
- one2code commented on: https://github.com/hackforla/website/pull/5158#issuecomment-1675654606 at 08/11/2023, 08:26 PM PDT
- one2code commented on: https://github.com/hackforla/website/pull/5183#issuecomment-1675654725 at 08/11/2023, 08:27 PM PDT
- one2code submitted review: https://github.com/hackforla/website/pull/5183#pullrequestreview-1576007852 at 08/13/2023, 08:25 PM PDT
- one2code submitted review: https://github.com/hackforla/website/pull/5170#pullrequestreview-1576046778 at 08/13/2023, 09:17 PM PDT
- one2code submitted review: https://github.com/hackforla/website/pull/5239#pullrequestreview-1581487358 at 08/16/2023, 02:58 PM PDT
- one2code commented on: https://github.com/hackforla/website/pull/5158#issuecomment-1681487627 at 08/16/2023, 06:46 PM PDT
- one2code commented on: https://github.com/hackforla/website/pull/5392#issuecomment-1700326637 at 08/30/2023, 08:47 PM PDT
- one2code commented on: https://github.com/hackforla/website/pull/5382#issuecomment-1700326905 at 08/30/2023, 08:47 PM PDT
- one2code submitted review: https://github.com/hackforla/website/pull/5392#pullrequestreview-1603891782 at 08/30/2023, 08:54 PM PDT
- one2code submitted review: https://github.com/hackforla/website/pull/5382#pullrequestreview-1603913574 at 08/30/2023, 09:07 PM PDT
- one2code assigned: https://github.com/hackforla/website/issues/3750#issuecomment-1597992124 at 08/30/2023, 09:13 PM PDT
- one2code commented on: https://github.com/hackforla/website/issues/3750#issuecomment-1700342026 at 08/30/2023, 09:14 PM PDT
- one2code commented on: https://github.com/hackforla/website/issues/3750#issuecomment-1718664451 at 09/13/2023, 07:41 PM PDT
- one2code submitted review: https://github.com/hackforla/website/pull/5522#pullrequestreview-1625750126 at 09/13/2023, 07:58 PM PDT
- one2code submitted review: https://github.com/hackforla/website/pull/5521#pullrequestreview-1625763782 at 09/13/2023, 08:21 PM PDT
- one2code submitted review: https://github.com/hackforla/website/pull/5564#pullrequestreview-1636840258 at 09/20/2023, 08:58 PM PDT
- one2code submitted review: https://github.com/hackforla/website/pull/5564#pullrequestreview-1636840495 at 09/20/2023, 08:59 PM PDT
- one2code commented on: https://github.com/hackforla/website/pull/6216#issuecomment-1925431149 at 02/03/2024, 11:06 AM PST
- one2code submitted review: https://github.com/hackforla/website/pull/6216#pullrequestreview-1861481163 at 02/04/2024, 08:27 AM PST
- one2code assigned: https://github.com/hackforla/website/issues/4505 at 02/04/2024, 07:47 PM PST
- one2code commented on: https://github.com/hackforla/website/issues/4505#issuecomment-1926183878 at 02/04/2024, 07:50 PM PST
- one2code commented on: https://github.com/hackforla/website/issues/4505#issuecomment-1936686524 at 02/09/2024, 02:27 PM PST
- one2code unassigned: https://github.com/hackforla/website/issues/4505 at 03/20/2024, 02:36 PM PDT

**Comment by HackforLABot** [comment]
## Activity Log: one2code

#####  ⚠ Important note: The bot updates this issue automatically - do not edit

- one2code opened issue: [4335](https://github.com/hackforla/website/issues/4335) at 2023-03-28 09:19 PM PDT
- one2code assigned to issue: [4335](https://github.com/hackforla/website/issues/4335) at 2023-04-06 01:57 PM PDT
- one2code assigned to issue: [4354](https://github.com/hackforla/website/issues/4354#issuecomment-1493389723) at 2023-04-13 09:08 PM PDT
- one2code unassigned from issue: [4354](https://github.com/hackforla/website/issues/4354#issuecomment-1507898629) at 2023-04-13 11:59 PM PDT
- one2code assigned to issue: [4380](https://github.com/hackforla/website/issues/4380) at 2023-04-14 12:08 AM PDT
- one2code assigned to issue: [3524](https://github.com/hackforla/website/issues/3524#issuecomment-1240946700) at 2023-04-14 11:57 AM PDT
- one2code unassigned from issue: [3524](https://github.com/hackforla/website/issues/3524#issuecomment-1240946700) at 2023-04-14 11:57 AM PDT
- one2code assigned to issue: [4383](https://github.com/hackforla/website/issues/4383#issuecomment-1497129548) at 2023-04-14 11:58 AM PDT
- one2code commented on issue: [4380](https://github.com/hackforla/website/issues/4380#issuecomment-1509915110) at 2023-04-15 10:56 AM PDT
- one2code opened pull request: [4520](https://github.com/hackforla/website/pull/4520) at 2023-04-15 12:51 PM PDT
- one2code unassigned from issue: [4383](https://github.com/hackforla/website/issues/4383#issuecomment-1509090621) at 2023-04-15 12:59 PM PDT
- one2code commented on pull request: [4520](https://github.com/hackforla/website/pull/4520#issuecomment-1510445913) at 2023-04-16 11:03 AM PDT
- one2code pull request merged: [4520](https://github.com/hackforla/website/pull/4520#event-9018434056) at 2023-04-16 02:34 PM PDT
- one2code assigned to issue: [4400](https://github.com/hackforla/website/issues/4400) at 2023-04-16 07:05 PM PDT
- one2code commented on issue: [4400](https://github.com/hackforla/website/issues/4400#issuecomment-1510589969) at 2023-04-16 07:07 PM PDT
- one2code opened pull request: [4528](https://github.com/hackforla/website/pull/4528) at 2023-04-16 09:30 PM PDT
- one2code pull request merged: [4528](https://github.com/hackforla/website/pull/4528#event-9031100177) at 2023-04-17 09:08 PM PDT
- one2code commented on issue: [4335](https://github.com/hackforla/website/issues/4335#issuecomment-1515612317) at 2023-04-19 07:09 PM PDT
- one2code assigned to issue: [3843](https://github.com/hackforla/website/issues/3843#issuecomment-1499866166) at 2023-04-20 06:25 AM PDT
- one2code commented on issue: [3843](https://github.com/hackforla/website/issues/3843#issuecomment-1516337227) at 2023-04-20 06:31 AM PDT
- one2code commented on issue: [3843](https://github.com/hackforla/website/issues/3843#issuecomment-1526904211) at 2023-04-27 07:47 PM PDT
- one2code closed issue as completed: [4335](https://github.com/hackforla/website/issues/4335#event-9120475701) at 2023-04-27 07:49 PM PDT
- one2code commented on issue: [3843](https://github.com/hackforla/website/issues/3843#issuecomment-1536768291) at 2023-05-05 01:52 PM PDT
- one2code commented on issue: [3843](https://github.com/hackforla/website/issues/3843#issuecomment-1542414695) at 2023-05-10 08:32 AM PDT
- one2code opened pull request: [4661](https://github.com/hackforla/website/pull/4661) at 2023-05-12 06:55 PM PDT
- one2code commented on pull request: [4661](https://github.com/hackforla/website/pull/4661#issuecomment-1550598025) at 2023-05-16 07:30 PM PDT
- one2code opened pull request: [4688](https://github.com/hackforla/website/pull/4688) at 2023-05-17 05:57 PM PDT
- one2code pull request closed w/o merging: [4661](https://github.com/hackforla/website/pull/4661#event-9274744149) at 2023-05-17 05:58 PM PDT
- one2code reopened pull request: [4661](https://github.com/hackforla/website/pull/4661#event-9274744149) at 2023-05-18 11:58 AM PDT
- one2code pull request closed w/o merging: [4661](https://github.com/hackforla/website/pull/4661#event-9282894173) at 2023-05-18 11:59 AM PDT
- one2code submitted pull request review: [4683](https://github.com/hackforla/website/pull/4683#pullrequestreview-1433774626) at 2023-05-18 09:56 PM PDT
- one2code commented on pull request: [4698](https://github.com/hackforla/website/pull/4698#issuecomment-1555961148) at 2023-05-20 10:51 AM PDT
- one2code submitted pull request review: [4698](https://github.com/hackforla/website/pull/4698#pullrequestreview-1435482769) at 2023-05-21 12:02 AM PDT
- one2code submitted pull request review: [4683](https://github.com/hackforla/website/pull/4683#pullrequestreview-1438593723) at 2023-05-22 10:49 PM PDT
- one2code commented on pull request: [4688](https://github.com/hackforla/website/pull/4688#issuecomment-1559420378) at 2023-05-23 06:51 AM PDT
- one2code commented on pull request: [4708](https://github.com/hackforla/website/pull/4708#issuecomment-1560059549) at 2023-05-23 01:10 PM PDT
- one2code submitted pull request review: [4708](https://github.com/hackforla/website/pull/4708#pullrequestreview-1440884350) at 2023-05-23 08:36 PM PDT
- one2code submitted pull request review: [4708](https://github.com/hackforla/website/pull/4708#pullrequestreview-1442719782) at 2023-05-24 01:42 PM PDT
- one2code submitted pull request review: [4725](https://github.com/hackforla/website/pull/4725#pullrequestreview-1442970195) at 2023-05-24 07:02 PM PDT
- one2code commented on pull request: [4726](https://github.com/hackforla/website/pull/4726#issuecomment-1562307936) at 2023-05-24 10:41 PM PDT
- one2code pull request merged: [4688](https://github.com/hackforla/website/pull/4688#event-9343513105) at 2023-05-25 11:57 AM PDT
- one2code opened pull request: [4733](https://github.com/hackforla/website/pull/4733) at 2023-05-25 07:46 PM PDT
- one2code pull request merged: [4733](https://github.com/hackforla/website/pull/4733#event-9346034481) at 2023-05-25 07:57 PM PDT
- one2code commented on pull request: [4747](https://github.com/hackforla/website/pull/4747#issuecomment-1565176270) at 2023-05-26 07:57 PM PDT
- one2code submitted pull request review: [4747](https://github.com/hackforla/website/pull/4747#pullrequestreview-1447134137) at 2023-05-26 08:27 PM PDT
- one2code submitted pull request review: [4747](https://github.com/hackforla/website/pull/4747#pullrequestreview-1447728590) at 2023-05-27 06:03 PM PDT
- one2code commented on pull request: [4726](https://github.com/hackforla/website/pull/4726#issuecomment-1565804611) at 2023-05-27 07:16 PM PDT
- one2code submitted pull request review: [4726](https://github.com/hackforla/website/pull/4726#pullrequestreview-1447805971) at 2023-05-27 07:46 PM PDT
- one2code assigned to issue: [3975](https://github.com/hackforla/website/issues/3975#issuecomment-1565195706) at 2023-05-27 08:26 PM PDT
- one2code commented on issue: [3975](https://github.com/hackforla/website/issues/3975#issuecomment-1565841497) at 2023-05-27 08:33 PM PDT
- one2code submitted pull request review: [4726](https://github.com/hackforla/website/pull/4726#pullrequestreview-1447849700) at 2023-05-27 08:52 PM PDT
- one2code commented on pull request: [4754](https://github.com/hackforla/website/pull/4754#issuecomment-1567910077) at 2023-05-30 12:29 AM PDT
- one2code submitted pull request review: [4754](https://github.com/hackforla/website/pull/4754#pullrequestreview-1450553645) at 2023-05-30 02:50 AM PDT
- one2code commented on pull request: [4767](https://github.com/hackforla/website/pull/4767#issuecomment-1573037525) at 2023-06-01 07:12 PM PDT
- one2code commented on issue: [3975](https://github.com/hackforla/website/issues/3975#issuecomment-1574323760) at 2023-06-02 02:15 PM PDT
- one2code submitted pull request review: [4767](https://github.com/hackforla/website/pull/4767#pullrequestreview-1464139771) at 2023-06-05 07:50 PM PDT
- one2code opened pull request: [4796](https://github.com/hackforla/website/pull/4796) at 2023-06-10 01:53 PM PDT
- one2code commented on pull request: [4817](https://github.com/hackforla/website/pull/4817#issuecomment-1588116510) at 2023-06-12 02:16 PM PDT
- one2code submitted pull request review: [4817](https://github.com/hackforla/website/pull/4817#pullrequestreview-1475931981) at 2023-06-12 03:52 PM PDT
- one2code submitted pull request review: [4817](https://github.com/hackforla/website/pull/4817#pullrequestreview-1476202063) at 2023-06-12 09:03 PM PDT
- one2code commented on pull request: [4833](https://github.com/hackforla/website/pull/4833#issuecomment-1590356151) at 2023-06-13 07:38 PM PDT
- one2code commented on pull request: [4832](https://github.com/hackforla/website/pull/4832#issuecomment-1590356456) at 2023-06-13 07:38 PM PDT
- one2code pull request merged: [4796](https://github.com/hackforla/website/pull/4796#event-9560042415) at 2023-06-17 08:18 PM PDT
- one2code submitted pull request review: [4833](https://github.com/hackforla/website/pull/4833#pullrequestreview-1484964312) at 2023-06-17 09:36 PM PDT
- one2code commented on pull request: [4832](https://github.com/hackforla/website/pull/4832#issuecomment-1596525663) at 2023-06-18 10:23 PM PDT
- one2code commented on pull request: [4832](https://github.com/hackforla/website/pull/4832#issuecomment-1598052022) at 2023-06-19 08:18 PM PDT
- one2code submitted pull request review: [4832](https://github.com/hackforla/website/pull/4832#pullrequestreview-1489332130) at 2023-06-20 05:49 PM PDT
- one2code commented on pull request: [4875](https://github.com/hackforla/website/pull/4875#issuecomment-1605670075) at 2023-06-24 10:57 AM PDT
- one2code submitted pull request review: [4875](https://github.com/hackforla/website/pull/4875#pullrequestreview-1496772952) at 2023-06-24 11:37 AM PDT
- one2code commented on pull request: [4878](https://github.com/hackforla/website/pull/4878#issuecomment-1605768079) at 2023-06-24 04:06 PM PDT
- one2code commented on pull request: [4879](https://github.com/hackforla/website/pull/4879#issuecomment-1605839876) at 2023-06-24 08:12 PM PDT
- one2code submitted pull request review: [4879](https://github.com/hackforla/website/pull/4879#pullrequestreview-1497088682) at 2023-06-25 07:02 AM PDT
- one2code submitted pull request review: [4878](https://github.com/hackforla/website/pull/4878#pullrequestreview-1497098445) at 2023-06-25 07:28 AM PDT
- one2code commented on pull request: [4882](https://github.com/hackforla/website/pull/4882#issuecomment-1607657503) at 2023-06-26 07:54 AM PDT
- one2code commented on pull request: [4879](https://github.com/hackforla/website/pull/4879#issuecomment-1608209327) at 2023-06-26 01:38 PM PDT
- one2code submitted pull request review: [4882](https://github.com/hackforla/website/pull/4882#pullrequestreview-1499914638) at 2023-06-26 09:22 PM PDT
- one2code submitted pull request review: [4879](https://github.com/hackforla/website/pull/4879#pullrequestreview-1499921835) at 2023-06-26 09:29 PM PDT
- one2code assigned to issue: [4475](https://github.com/hackforla/website/issues/4475#issuecomment-1608566987) at 2023-06-27 02:14 AM PDT
- one2code commented on issue: [4475](https://github.com/hackforla/website/issues/4475#issuecomment-1609113855) at 2023-06-27 02:18 AM PDT
- one2code commented on issue: [4475](https://github.com/hackforla/website/issues/4475#issuecomment-1610674971) at 2023-06-27 09:17 PM PDT
- one2code opened pull request: [4896](https://github.com/hackforla/website/pull/4896) at 2023-06-28 12:21 AM PDT
- one2code submitted pull request review: [4832](https://github.com/hackforla/website/pull/4832#pullrequestreview-1505380205) at 2023-06-29 07:07 AM PDT
- one2code commented on pull request: [4899](https://github.com/hackforla/website/pull/4899#issuecomment-1613266890) at 2023-06-29 07:19 AM PDT
- one2code commented on pull request: [4896](https://github.com/hackforla/website/pull/4896#issuecomment-1613286514) at 2023-06-29 07:30 AM PDT
- one2code commented on pull request: [4902](https://github.com/hackforla/website/pull/4902#issuecomment-1615352610) at 2023-06-30 06:59 PM PDT
- one2code submitted pull request review: [4879](https://github.com/hackforla/website/pull/4879#pullrequestreview-1509648034) at 2023-07-02 10:50 AM PDT
- one2code commented on pull request: [4905](https://github.com/hackforla/website/pull/4905#issuecomment-1620784724) at 2023-07-04 02:59 PM PDT
- one2code commented on pull request: [4896](https://github.com/hackforla/website/pull/4896#issuecomment-1620955932) at 2023-07-04 08:16 PM PDT
- one2code commented on pull request: [4896](https://github.com/hackforla/website/pull/4896#issuecomment-1622949664) at 2023-07-05 09:17 PM PDT
- one2code commented on pull request: [4896](https://github.com/hackforla/website/pull/4896#issuecomment-1626355160) at 2023-07-07 04:22 PM PDT
- one2code commented on pull request: [4896](https://github.com/hackforla/website/pull/4896#issuecomment-1627576891) at 2023-07-08 07:22 PM PDT
- one2code submitted pull request review: [4905](https://github.com/hackforla/website/pull/4905#pullrequestreview-1520946179) at 2023-07-09 12:00 PM PDT
- one2code commented on pull request: [4896](https://github.com/hackforla/website/pull/4896#issuecomment-1629894651) at 2023-07-10 05:00 PM PDT
- one2code commented on pull request: [4936](https://github.com/hackforla/website/pull/4936#issuecomment-1631745229) at 2023-07-11 07:12 PM PDT
- one2code pull request merged: [4896](https://github.com/hackforla/website/pull/4896#event-9795862162) at 2023-07-12 01:03 AM PDT
- one2code commented on pull request: [4927](https://github.com/hackforla/website/pull/4927#issuecomment-1635212536) at 2023-07-13 08:24 PM PDT
- one2code submitted pull request review: [4927](https://github.com/hackforla/website/pull/4927#pullrequestreview-1529557337) at 2023-07-13 08:59 PM PDT
- one2code submitted pull request review: [4936](https://github.com/hackforla/website/pull/4936#pullrequestreview-1531302726) at 2023-07-14 11:39 PM PDT
- one2code commented on pull request: [4936](https://github.com/hackforla/website/pull/4936#issuecomment-1637346041) at 2023-07-16 09:05 PM PDT
- one2code submitted pull request review: [4936](https://github.com/hackforla/website/pull/4936#pullrequestreview-1531905918) at 2023-07-16 09:08 PM PDT
- one2code commented on pull request: [4953](https://github.com/hackforla/website/pull/4953#issuecomment-1640893749) at 2023-07-18 12:38 PM PDT
- one2code submitted pull request review: [4953](https://github.com/hackforla/website/pull/4953#pullrequestreview-1536212764) at 2023-07-18 08:55 PM PDT
- one2code commented on pull request: [5006](https://github.com/hackforla/website/pull/5006#issuecomment-1643056606) at 2023-07-19 08:41 PM PDT
- one2code submitted pull request review: [5006](https://github.com/hackforla/website/pull/5006#pullrequestreview-1538379206) at 2023-07-19 08:50 PM PDT
- one2code submitted pull request review: [5006](https://github.com/hackforla/website/pull/5006#pullrequestreview-1540125617) at 2023-07-20 02:35 PM PDT
- one2code submitted pull request review: [5105](https://github.com/hackforla/website/pull/5105#pullrequestreview-1551301245) at 2023-07-27 08:52 PM PDT
- one2code submitted pull request review: [4953](https://github.com/hackforla/website/pull/4953#pullrequestreview-1557024399) at 2023-08-01 06:35 AM PDT
- one2code submitted pull request review: [5144](https://github.com/hackforla/website/pull/5144#pullrequestreview-1564070057) at 2023-08-05 10:42 PM PDT
- one2code submitted pull request review: [5170](https://github.com/hackforla/website/pull/5170#pullrequestreview-1570962630) at 2023-08-09 08:42 PM PDT
- one2code commented on pull request: [5158](https://github.com/hackforla/website/pull/5158#issuecomment-1675654606) at 2023-08-11 08:26 PM PDT
- one2code commented on pull request: [5183](https://github.com/hackforla/website/pull/5183#issuecomment-1675654725) at 2023-08-11 08:27 PM PDT
- one2code submitted pull request review: [5183](https://github.com/hackforla/website/pull/5183#pullrequestreview-1576007852) at 2023-08-13 08:25 PM PDT
- one2code submitted pull request review: [5170](https://github.com/hackforla/website/pull/5170#pullrequestreview-1576046778) at 2023-08-13 09:17 PM PDT
- one2code submitted pull request review: [5239](https://github.com/hackforla/website/pull/5239#pullrequestreview-1581487358) at 2023-08-16 02:58 PM PDT
- one2code commented on pull request: [5158](https://github.com/hackforla/website/pull/5158#issuecomment-1681487627) at 2023-08-16 06:46 PM PDT
- one2code commented on pull request: [5392](https://github.com/hackforla/website/pull/5392#issuecomment-1700326637) at 2023-08-30 08:47 PM PDT
- one2code commented on pull request: [5382](https://github.com/hackforla/website/pull/5382#issuecomment-1700326905) at 2023-08-30 08:47 PM PDT
- one2code submitted pull request review: [5392](https://github.com/hackforla/website/pull/5392#pullrequestreview-1603891782) at 2023-08-30 08:54 PM PDT
- one2code submitted pull request review: [5382](https://github.com/hackforla/website/pull/5382#pullrequestreview-1603913574) at 2023-08-30 09:07 PM PDT
- one2code assigned to issue: [3750](https://github.com/hackforla/website/issues/3750#issuecomment-1597992124) at 2023-08-30 09:13 PM PDT
- one2code commented on issue: [3750](https://github.com/hackforla/website/issues/3750#issuecomment-1700342026) at 2023-08-30 09:14 PM PDT
- one2code commented on issue: [3750](https://github.com/hackforla/website/issues/3750#issuecomment-1718664451) at 2023-09-13 07:41 PM PDT
- one2code submitted pull request review: [5522](https://github.com/hackforla/website/pull/5522#pullrequestreview-1625750126) at 2023-09-13 07:58 PM PDT
- one2code submitted pull request review: [5521](https://github.com/hackforla/website/pull/5521#pullrequestreview-1625763782) at 2023-09-13 08:21 PM PDT
- one2code submitted pull request review: [5564](https://github.com/hackforla/website/pull/5564#pullrequestreview-1636840258) at 2023-09-20 08:58 PM PDT
- one2code submitted pull request review: [5564](https://github.com/hackforla/website/pull/5564#pullrequestreview-1636840495) at 2023-09-20 08:59 PM PDT
- one2code commented on pull request: [6216](https://github.com/hackforla/website/pull/6216#issuecomment-1925431149) at 2024-02-03 11:06 AM PST
- one2code submitted pull request review: [6216](https://github.com/hackforla/website/pull/6216#pullrequestreview-1861481163) at 2024-02-04 08:27 AM PST
- one2code assigned to issue: [4505](https://github.com/hackforla/website/issues/4505#issuecomment-1822065293) at 2024-02-04 07:47 PM PST
- one2code commented on issue: [4505](https://github.com/hackforla/website/issues/4505#issuecomment-1926183878) at 2024-02-04 07:50 PM PST
- one2code commented on issue: [4505](https://github.com/hackforla/website/issues/4505#issuecomment-1936686524) at 2024-02-09 02:27 PM PST
- one2code unassigned from issue: [4505](https://github.com/hackforla/website/issues/4505#issuecomment-2010678427) at 2024-03-20 02:36 PM PDT
